index.html

```diff
@@ -12,37 +12,111 @@
-                </section>
-            </div>
-
-            <div class="contact">
-                <p>
-                    This project is licensed under the BSD-2-Clause license,
-                    and is avilable on <a href="https://github.com/GMOD/jbrowse-registry">GitHub</a>.
-                    Portions of this project was supported by the National
-                    Science Foundation under Grant Number 1565146
-                </p>
+                <div class="intro-text">
+                   <div class="intro-heading">The GMOD Plugin Directory</div>
+                   <a style="color:white" href="{{ url }}">Version {{ version }}</a>
+                </div>
             </div>
+        </header>
+        <div class="container">
+            <section id="app">
+                <ul class="nav nav-pills">
+                    <li class="active">
+                        <a>Plugins</a>
+                    </li>
+                    <li>
+                        <a href="https://github.com/GMOD/jbrowse-registry#registering-a-plugin" target="_blank">Register a Plugin</a>
+                    </li>
+                    <li>
+                        <a href="https://github.com/GMOD/jbrowse-registry/pulls?utf8=%E2%9C%93&q=is%3Apr%20label%3Anew-plugin%20" target="_blank">Pending plugins</a>
+                    </li>
+                </ul>
+                <nav class="navbar navbar-default">
+                    <div class="container-fluid">
+                        <div class="input-group">
+                            <input type="text" class="form-control" placeholder="Filter plugins..." ng-change="termChanged()"  ng-model="searchTerm" aria-describedby="plug-addon">
+                            <span id="plug-addon" class="input-group-addon glyphicon glyphicon-search" aria-hidden="true"></span>
+                        </div>
+                        <div class="filter-group">
+                           <form class="form-inline filter-form" role="form">
+                                <div class="form-group">
+                                    <label>Sort By:</label>
+                                    <select class="form-control" ng-model="sortKey">
+                                        <option value="name" selected>Name</option>
+                                        <option value="author">Author</option>
+                                        <option value="added">Date added</option>
+                                        <option value="license">License</option>
+                                    </select>
+                                </div>
+                                <div class="form-group">		
+                                    <label>Order:</label>		
+                                    <select class="form-control" ng-model="order">		
+                                        <option value= '+' selected>Ascending</option>		
+                                        <option value= '-'>Descending</option>		
```

Comment:
got eslint config added in this PR to check style/correctness but looks like it didn't catch this. would be nice to get some good autoformatting